app/backend/approaches/chatreadretrieveread.py

```diff
@@ -30,26 +30,26 @@ class ChatReadRetrieveReadApproach(Approach):
     Try not to repeat questions that have already been asked.
     Only generate questions and do not generate any text before or after the questions, such as 'Next Questions'"""
 
-    query_prompt_template = """Below is a history of the conversation so far, and a new question asked by the user that needs to be answered by searching in a knowledge base about employee healthcare plans and the employee handbook.
-    Generate a search query based on the conversation and the new question. 
+    query_prompt_template = """Assistant is intelligent bot that helps to generate a search query.
```

Comment:
@pamelafox  @chuwik  I tested for sometime, search query generation seems to be in parity with new template and fewer shots. It would be helpful if you play with my branch in local to see if quality is stable.